src/express-server/middlewares/parsedQuery.ts

```diff
@@ -0,0 +1,7 @@
+import querystring from "querystring";
+import url from "url";
+
+export const parsedQuery = (req: any, res: any, next: Function): void  => {
+  req.parsedQuery = querystring.parse(url.parse(req.url).query);
```

Comment:
please make it manually, without using 3rdparty libs, it's easily accessible through `req.query`